tests/golden/generic.expected

```diff
@@ -10,7 +10,7 @@ Error in $: parsing Types.SomeType(Unary) failed, key "contents" not present
 Error in $: parsing Types.SomeType(Record) failed, key "testone" not present
 Error in $.testone: parsing Double failed, unexpected Boolean
 Error in $.tag: parsing Types.SomeType failed, expected tag field to be one of ["nullary","unary","product","record","list"], but found tag "X"
-Error in $: parsing Types.SomeType failed, key "tag" not present
+Error in $: parsing Types.SomeType failed, expected an object with key "tag" where "tag" is associated to one of ["nullary","unary","product","record","list"]. The parser returned error was: key "tag" not present
```

Comment:
- "expected Object" instead of "expected an object" would make the error look more uniform with the rest.
- "with key "tag" associated to" instead of "with key "tag" where "tag" is associated to"
- you can also remove "the parser returned error was"

Reply:
and I think "not found" sounds better than "not present"